<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fire Department Management System</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actSearch"/>
    <addaction name="separator"/>
    <addaction name="actFirefighterAdd"/>
    <addaction name="separator"/>
    <addaction name="actActive_Drill"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actFirefighterAdd">
   <property name="text">
    <string>Firefighter Add</string>
   </property>
   <property name="toolTip">
    <string>Firefighter Add</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actActive_Drill">
   <property name="text">
    <string>Active Drill</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
